src/factories/Container.ts

```diff
@@ -49,10 +51,10 @@ module n3Charts.Factory {
       var {left, top} = event.currentTarget.getBoundingClientRect();
 
       var xScale = this.factoryMgr.get('x-axis').scale;
-      var x = xScale.invert(event.x - left - dim.margin.left);
+      var x = xScale.invert(event.clientX - left - dim.margin.left);
```

Comment:
Sure this is `clientX`? Does this work when you have multiple charts? I think you should use `offsetX`, but be careful in FF http://stackoverflow.com/a/11334572